bigquery_etl/stripe/__init__.py

```diff
@@ -166,6 +190,10 @@ def import_(api_key, date, table, file, resource):
                 job.result()
             except Exception as e:
                 print(f"{job.job_id} failed: {e}", file=sys.stderr)
+                for error in job.errors or ():
+                    message = error.get("message")
+                    if message and message != getattr(e, "message", None):
+                        print(message, file=sys.stderr)
```

Comment:
the extra lines here include information like what field is missing from the schema, and the row at which the job failed.